### ISSUE TITLE
telnyx-mock fails with latest openapi spec

### ISSUE THREAD

**Issue by ledbettj** [team-telnyx/telnyx-mock#19]
Introduced by https://github.com/team-telnyx/openapi/commit/425decf246f260dc1df896dbd81585d8f9ccd868

```
$./telnyx-mock -verbose -spec-skip-cache
telnyx-mock 0.8.10
Downloading API spec file from: https://raw.githubusercontent.com/team-telnyx/openapi/master/openapi/spec3.json
error decoding spec: unsupported field in JSON schema: 'uniqueItems'
```

Running with the previous commits works as expected.

**Comment by d-telnyx** [comment]
@ledbettj Thank you for opening the issue! I've got the fix opened down stream and waiting on the SDLC triggers to get it patched ASAP

**Comment by d-telnyx** [comment]
solved in release 0.8.12
